doc/source/starting.rst

```diff
@@ -423,6 +436,53 @@ This previous code is much simpler now:
     for surface in cell.surfaces.pz:
         surface.location += 10
 
+Setting Boundary Conditions
```

Comment:
This section is helpful and contains helpful clarifying additions. Thank you for including it!